docs/user-guide/nodes/configuring-nodes.md

```diff
@@ -87,8 +85,6 @@ private_key = certs/cert_node_73768d5f-6a66-47de-8533-1291c4ef59d1/MPSPDZ_certif
 public_key = certs/cert_node_73768d5f-6a66-47de-8533-1291c4ef59d1/MPSPDZ_certificate.pem
 mpspdz_ip = localhost
 mpspdz_port = 14004
-allow_default_biprimes = True
```

Comment:
Thank you for searching the doc for obsoleted references to biprime. I found a few more to be updated
- docs/tutorials/security/secure-aggregation.ipynb references `exp.secagg.biprime` which does not exist anymore
- docs/user-guideresearcher/experiment.md details in json format the content of the train request listing a `secagg_biprime_id` which does not exist (this doc was already not up to date as the 4 secagg_* were replaced with `secagg_arguments` already )
- docs/user-guide/secagg/introduction.md explains generation of biprime which was removed
- docs/user-guide/secagg/researcher-interface.md mentions generation of biprime
- tests/test_message.py has a test with a secagg_biprime_id argument (which is correct for a unit test but might now be confusing, we can choose to remove this line)

Reply:
Thank you, I updated those appearances.